frontend/main-app/src/app/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "@digdir/design-system-tokens/brand/digdir/tokens.css";
+import classes from "./App.module.css";
+import { Header } from "../components/Header/Header";
```

Comment:
If you add an ```index.ts``` file inside the component (Header in this case), then you can change this line to the following: 
```suggestion
import { Header } from "../components/Header";
```

The ```index.ts``` file should then only include what you export and look something like this then: 
```ts
export { Header } from './Header'
```
😄

Reply:
This goes for all the components that are put inside their own folders. 
I really like that you structure the components into one folder for each component! 😄

Reply:
That's a neat trick, thanks! 😄